src/components/avatars/OcAvatarItem.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <span
+    class="oc-avatar-item"
+    :style="{ backgroundColor: backgroundColor, '--icon-color': iconColor }"
+    :aria-label="accessibleLabel === '' ? null : accessibleLabel"
+    :aria-hidden="accessibleLabel === '' ? 'true' : null"
+    :focusable="accessibleLabel === '' ? 'false' : null"
+    :role="accessibleLabel === '' ? null : 'img'"
+    :data-test-link-name="name"
```

Comment:
Please rename the prop to `data-test-item-name`. It's not about just links anymore. ;-)